### DISCUSSION TITLE
[Feature Request] Add the new tab button back to the "customize toolbar" page

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#404] [Feature Request]
### Is your feature request related to a problem?

I don't like how the new tab button is at the bottom of the vertical tabs list, it makes it time consuming to access.

### Describe the solution you'd like

Firefox has a new tab button which can be placed anywhere on the url or bookmarks bar. If its not placed then it appears in the "customize toolbar" section like so (ignore the large size, that's my CSS theme):

<img width="309" alt="Screenshot 2023-08-23 at 5 35 49 AM" src="https://github.com/Floorp-Projects/Floorp/assets/42646062/99b01ee5-40e0-4c94-9f3b-a19f19a88311">

**Comment by menndouyukkuri** [comment]
The vertical tab seems to use a sidebar, so even if there is a new tab button on the toolbar, I think it will not be possible to move it to the vertical tab bar.

**Comment by YousufSSyed** [comment]
I made another feature request for that https://github.com/Floorp-Projects/Floorp/issues/398

**Comment by menndouyukkuri** [comment]
This has been resolved in 11.5.0.
